### PR TITLE
Correctly write CONECT record

### PR DESCRIPTION
**Pull request by hainm**

### CODE REVIEW

File: pdb4amber/pdb4amber.py [hunk 1]
```diff
@@ -598,6 +598,7 @@ def run(
     else:
         # keep all models
         write_kwargs = dict()
+    write_kwargs['increase_tercount'] = False # so CONECT record can work properly
```
Comment (dkoes) [inline pdb4amber/pdb4amber.py]: `TypeError: write() got an unexpected keyword argument 'increase_tercount'` 
Does this fix require additional changes to parmed that haven't been pushed?  
I'd like to port the fix...

Comment (hainm) [reply pdb4amber/pdb4amber.py]: hi @dkoes, you need to install ParmEd >= 3.0: https://github.com/ParmEd/ParmEd/releases/tag/3.0.0

The update is here: https://github.com/ParmEd/ParmEd/pull/918

Comment (dkoes) [reply pdb4amber/pdb4amber.py]: Thanks! I was looking at the wrong github for parmed.  I've got it working now.

Comment (hainm) [reply pdb4amber/pdb4amber.py]: cc @swails: can you update amber-md/parmed? thanks.

Comment (swails) [reply pdb4amber/pdb4amber.py]: Done.

**Pull request merged**